lib/src/common/parameters/excluded_identifiers_list_parameter.dart

```diff
@@ -26,6 +26,39 @@ class ExcludedIdentifiersListParameter {
     for (final item in excludeList) {
       if (item is Map) {
         exclude.add(ExcludedIdentifierParameter.fromJson(item));
+      } else if (item is String) {
+        exclude.add(
+          ExcludedIdentifierParameter(
+            methodName: item,
+          ),
+        );
+      }
```

Comment:
For plain Strings -- I think it can be both classes _and_ methods.
Add a separate parameter, `declarationName` for such cases?

While we're at it, I think we should be careful about these exclusions being noo unspecific.

For example, I think the behavior should be like this:

```dart
exclude:
  - excluded

// lint
void excluded() {}

class A {
  // no lint
  void excluded() {}
}
```

Reply:
Discussed separately -- it makes sense to keep. the current behavior.